src/decaylanguage/dec/dec.py

```diff
@@ -131,8 +131,8 @@ def __init__(self, *filenames):
             # Content of input file(s)
             self._dec_file = stream.read()
         else:
-            self._dec_file_names = None
-            self._dec_file = None
+            self._dec_file_names = None  # type: ignore
+            self._dec_file = None  # type: ignore
```

Comment:
Don't normally do this, but it was faster/easier for me not to look up or guess the types. We should type particle first.